tests/test_api.py

```diff
@@ -12,24 +16,24 @@ def test_api(client):
     resp = client.get('/api/')
     assert resp.status_code == 200
 
-def test_resource_one(client):
-    resp = client.get('/api/resource/one')
+def test_vote_count(client):
+    resp = client.get('/api/vote/count')
     assert resp.status_code == 200
 
-def test_resource_one_post(client):
-    resp = client.post('/api/resource/one')
+def test_vote_post(client):
+    resp = client.post('/api/vote')
```

Comment:
This test is failing for me, presumably because we are not providing any data in the post call.

The API as such works when casting vote.

Reply:
Fixed, with a small initial test fixture.